README.md

```diff
@@ -1,9 +1,23 @@
 # ansible-gunicorn
 
-An Ansible role for installing and configuring gunicorn.
+An Ansible role for installing and configuring gunicorn. Provides the `Restart gunicorn` handler to reload gunicorn when you install/update your app.
 
 ## Role Variables
 
+- `develop` - is a develop environment. If set, will reload after each request (default: `false`)
```

Comment:
These variables seem redundant if all it is doing is flipping `reload` variables and aren't really namespaced (could cause unexpected behavior). I would axe these and let roles/playbooks that inherit/use this role handle when reload should be flipped, especially since there could be cases when someone would want develop/test environments to more closely mimic production settings.

Also - not sure this is actually being used for what the comment describes based on the code below.